core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -164,7 +164,11 @@
             ng-mouseleave="onMouseoutDropdownMenu($event)">
           <li>
             <a href style="padding: 0; width: 200px;" ng-click="onLoginButtonClicked()">
-              <img ng-src="<[getStaticImageUrl('/signin/Red-signin-Long-base-44dp.png')]>" alt="Sign in with Google">
+              <div class="oppia-signin-google">
+                <i class="oppia-signin-google-icon fa fa-google-plus"></i>
```

Comment:
Interestingly, I can't even get to that button from the screenreader (related to another documented issue). So the screenreader doesn't see it at all right now and simply uses the sign in button to sign in.

Everything else LGTM!